lib/logger.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
+
+"use strict";
+
+const LEVELS = {
+  info: 10,
+  warn: 20,
+  error: 30
+};
+
+let level = LEVELS.info;
+
+const logger = {
+
+  setLevel(newLevel) {
+    const lowerLevel = (newLevel || "info").toLowerCase();
+    const numericLevel = LEVELS[lowerLevel];
+
+    if (!numericLevel) {
+      throw new Error(
+        `Log level must be one of ${Object.keys(LEVELS).join(", ")}. Received "${newLevel}".`);
+    }
+
+    level = numericLevel;
+  },
+
+  info(...message) {
+    if (level <= LEVELS.info) {
+      console.info(...message);
+    }
+  },
+
+  warn(...message) {
+    if (level <= LEVELS.warn) {
+      console.warn(...message);
+    }
+  },
+
+  error(...message) {
```

Comment:
there are still Node 4 usage of electrode-server.  Could you please avoid using the spread operators?  Otherwise this will be a major bump.

Reply:
It does sound like it would be a problem!  Fixed